README.md

```diff
@@ -42,9 +42,10 @@ Currently the following variables are supported:
 * `molecule_openstack_ci_security_group_name` - SSH keypair name prefix to use
  when generating keypair. Defaults to "molecule_key". The actual keypair
  created will have the run hash appended to its name.
-* `molecule_openstack_ci_cloud` - name of the OpenStack cloud to connect to. This defaults to being omitted,
- in which case the openstack libraries will read the environment variable "OS\_CLOUD" by default. Set this
- value if you want to override that for specific cases or do not have the value set.
+* `molecule_openstack_ci_cloud` - name of the OpenStack cloud to connect to. This defaults to the value
```

Comment:
This, by the way, is how I actually ended up here. Despite having `OS_CLOUD` set, os_server (and its underlying machinery) was not detecting the default cloud, and it seemed odd to be carrying around the `{{ lookup('env', 'OS_CLOUD') | default('default', true) }}` from role to role. The simpler and more user-friendly approach taken by `molecule_docker_ci` was an inspiration for making this simpler.